platform/broadcom/sai.mk

```diff
@@ -4,6 +4,11 @@ BRCM_SAI_DEV = libsaibcm-dev_4.3.3.5-1_amd64.deb
 $(eval $(call add_derived_package,$(BRCM_SAI),$(BRCM_SAI_DEV)))
 $(BRCM_SAI_DEV)_URL = "https://sonicstorage.blob.core.windows.net/packages/bcmsai/4.3/master/libsaibcm-dev_4.3.3.5-1_amd64.deb?sv=2019-12-12&st=2021-04-26T22%3A17%3A56Z&se=2030-04-27T22%3A17%3A00Z&sr=b&sp=r&sig=2eWNoczh9P5c3Ir%2B1YL08ZkPol9GUkvNJKFGf%2BacXlI%3D"
```

Comment:
Dont you need to use correct -dev package as well? Here its still referencing 4.3 version of headers of BCMSAI?

Reply:
We have a different PR https://github.com/Azure/sonic-buildimage/pull/7753, which moves to the sai-dev 5.0.0.1. It will get updated.
But currently I have verified with 4.3 -dev package, and it works well as it is also based on SAI 1.8 headers.

Reply:
already updated with 5.0.0.1 to match XGS ASIC family.